graphql-dgs-spring-graphql/src/main/kotlin/com/netflix/graphql/dgs/springgraphql/autoconfig/DgsSpringGraphQLConfigurationProperties.kt

```diff
@@ -16,18 +16,13 @@
 
 package com.netflix.graphql.dgs.springgraphql.autoconfig
 
-import com.netflix.graphql.dgs.springgraphql.autoconfig.DgsSpringGraphQLConfigurationProperties.Companion.PREFIX
 import org.springframework.boot.context.properties.ConfigurationProperties
 import org.springframework.boot.context.properties.bind.DefaultValue
 
-@ConfigurationProperties(prefix = PREFIX)
+@ConfigurationProperties(prefix = "dgs.graphql.spring")
```

Comment:
Why do these need to change?

Reply:
These top level constants don't work when compiling with modules. I think this is a Kotlin compiler bug, but you get an error that the package where the constant is defined isn't exported to the module it's being referenced, however, it's the same module.